CONTRIBUTING.rst

```diff
@@ -119,6 +119,12 @@ Please do not use `toolbx <https://github.com/containers/toolbox>`_ or `distrobo
 
 Follow these steps to build the ISO you need.
 
+**NOTE: If you are targetting a RHEL branch, build the RPM container first:**::
+
+  make -f ./Makefile.am container-rpms-scratch
```

Comment:
I guess this should be 

```
make -f ./Makefile.am anaconda-rpm-build
```

Reply:
Indeed! To much (or too little) copy & paste. :P Fixed now.